pkg/executor/nginx/nginx.go

```diff
@@ -193,6 +201,11 @@ func mapDataDescToTemplateInput(openshiftConfig OpenshiftConfig) (*executor.Temp
 	nginxGzipForTemplate := nginxGzipMapToString(openshiftConfig.Web.Gzip)
 	nginxLocationForTemplate := nginxLocationsMapToString(openshiftConfig.Web.Locations, documentRoot, path)
 
+	notServingOnRoot := true
```

Comment:
Denne kan flyttes opp i blokken hvor du sjekket for path. Slipper du en ekstra sjekk.